terraform/custom_domains/environment_domains/workspace_variables/find_production_aks.tfvars.json

```diff
@@ -12,6 +12,19 @@
       "environment_short": "pd",
       "origin_hostname": "publish-production.teacherservices.cloud",
       "null_host_header": true
+    },
+    "find-teacher-training-courses.service.gov.uk": {
+      "front_door_name": "s189p01-fttc-svc-domains-fd",
+      "resource_group_name": "s189p01-fttdomains-rg",
+      "domains": [
+        "www"
+      ],
+      "cached_paths": [
+        "/assets/*"
+      ],
+      "environment_short": "pd",
+      "origin_hostname": "publish-production.teacherservices.cloud",
+      "null_host_header": true
```

Comment:
If true, then we need to add a new ingress *find-teacher-training-courses.service.gov.uk*
Can we try without it so it uses *publish-production.teacherservices.cloud*?